engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/text/util/TypeToDisplayTextNode.java

```diff
@@ -1,17 +1,18 @@
 package org.enso.interpreter.node.expression.builtin.text.util;
 
+import org.enso.interpreter.node.expression.builtin.meta.TypeOfNode;
+import org.enso.interpreter.runtime.callable.atom.Atom;
+import org.enso.interpreter.runtime.callable.atom.AtomConstructor;
+import org.enso.interpreter.runtime.data.Type;
+import org.enso.interpreter.runtime.type.TypesGen;
```

Comment:
VSCode (with Apache NetBeans based plugin) is configured to clean imports aggressively. For example it expands `*` imports to individual ones, removes unnecessary imports, etc. However the order is then finally set by `javafmtAll`. 

> I have seen a lot of import reordering in your recent PRs. 

Yeah, I've noticed that myself as well.

> on the same page and ensure whose setup is the preferred one.

`javafmtAll` is the final check, however it may be too _"forgiving"_. I can change my settings to instruct the IDE to not touch imports automatically.